File: Backend/ItHappened/ItHappenedDomain/Domain/Event.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ItHappenedDomain.Domain
{
  public class Event
  {
    public string eventId { set; get; }
    public string trackingId { set; get; }
    public DateTimeOffset eventDate { set; get; }
    public double? scale { set; get; }
    public Rating rating { set; get; }
    public string comment { set; get; }
    public DateTimeOffset dateOfChange { set; get; }
    public bool isDeleted { set; get; }
    private double longitude;
    private double lotitude;

    public double Lotitude
    {
      get => lotitude;
      set => lotitude = value;
    }
    public double Longitude
    {
      get => longitude;
      set => longitude = value;
    }
    

    public Event(string eventId, string trackingId,
      DateTimeOffset eventDate, 
      double? scale, Rating rating, string comment, 
      DateTimeOffset dateOfChange, bool isDeleted,
      double longitude, double lotitude)
    {
      this.eventId = eventId;
      this.trackingId = trackingId;
      this.eventDate = eventDate;
      this.scale = scale;
      this.rating = rating;
      this.comment = comment;
      this.dateOfChange = dateOfChange;
      this.isDeleted = isDeleted;
      this.lotitude = lotitude;
      this.longitude = longitude;
    }
  }
}

File: Android/ItHappenedAndroid/app/src/main/java/ru/lod_misis/ithappened/Domain/Event.java
package ru.lod_misis.ithappened.Domain;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.Calendar;
import java.util.Date;
import java.util.TimeZone;
import java.util.UUID;

import io.realm.RealmObject;
import io.realm.annotations.Index;
import io.realm.annotations.PrimaryKey;
import io.realm.annotations.Required;

/**
 * Created by Ded on 07.06.2018.
 */

public class Event extends RealmObject {

    public Event(UUID eventId, UUID trackingID, Date date, Double scale, Rating rating, String comment,Double lotitude,Double longitude)
    {
        this.eventId = eventId.toString();
        this.dateOfChange = Calendar.getInstance(TimeZone.getDefault()).getTime();
        this.scale = scale;
        this.rating = rating;
        this.comment = comment;
        this.lotitude=lotitude;
        this.longitude=longitude;
        this.trackingId = trackingID.toString();
        this.eventDate = date;
    }

    public Event(){}

    @Expose
    @SerializedName("eventId")
    public String eventId;
    @Expose
    @SerializedName("trackingId")
    public String trackingId;
    @Expose
    @SerializedName("eventDate")
    public Date eventDate;
    @Expose
    @SerializedName("scale")
    public Double scale;
    @Expose
    @SerializedName("rating")
    public Rating rating;
    @Expose
    @SerializedName("comment")
    public String comment;
    public Double lotitude;
    public Double longitude;
    @Expose
    @SerializedName("dateOfChange")
    public Date dateOfChange;
    @Expose
    @SerializedName("isDeleted")
    public boolean isDeleted = false;
}
